process/smartContract/process.go

```diff
@@ -908,7 +909,7 @@ func (sc *scProcessor) isSCExecutionAfterBuiltInFunc(
 			CallValue:      big.NewInt(0),
 			CallType:       callType,
 			GasPrice:       vmInput.GasPrice,
-			GasProvided:    vmOutput.GasRemaining - vmInput.GasLocked,
+			GasProvided:    scExecuteOutTransfer.GasLimit,
```

Comment:
is this backwards compatible?

Reply:
Yes it is. This feature is under built in functions

Reply:
ok